digitalstrategie/settings/base.py

```diff
@@ -86,6 +86,9 @@
                 'django.contrib.messages.context_processors.messages',
                 'wagtail.contrib.settings.context_processors.settings',
             ],
+            'libraries': {
+                'custom_filters': 'apps.contrib.templatetags.custom_filters'
+            }
```

Comment:
You shouldn't need to add it here. Just adding the templatetag into the templatetag folder (When there is an `__init__.py` in it as well) and registering it (as you did) should be enough!